nerdtribe/mr-binge
# TSLint to ESLint Updating

## Explanation of work done and why:


Tackled the recommended tslint to eslint upgrade.

## How to (manually) test:

1. Fetch `git fetch && git checkout package-updating origin/package-updating`
2. Purge:
- Delete node modules
- Clear cache `yarn cache clear`
- Install packages `yarn`
3. Verify no install issues.
4. Run program `yarn electron:serve`
5. New tab, run testing `yarn test:unit`
6. Verify no errors in either console

## Checklist:
Closes:
